src/renderer/lib/hooks/useAppListeners.ts

```diff
@@ -229,7 +229,19 @@ export const useAppListeners = () => {
 
   const setDolphinOpen = useDolphinStore((store) => store.setDolphinOpen);
   ipc_dolphinClosedEvent.renderer!.useEvent(
-    async ({ dolphinType }) => {
+    async ({ dolphinType, exitCode }) => {
+      if (exitCode !== 0) {
```

Comment:
we should handle the 2 known exit codes (0xC0000135/3221225781 for vcr and 3 for dolphin updates) with this case being the final `else if`.

Reply:
Yeah this is what I mentioned in the improvements section. I'll see about putting links in the toaster.